testing.rst

```diff
@@ -985,13 +985,19 @@ Response Assertions
     Asserts the response format returned by the
     :method:`Symfony\\Component\\HttpFoundation\\Response::getFormat` method
     is the same as the expected value.
-``assertResponseIsUnprocessable(string $message = '', bool $verbose = true)``
+``assertResponseIsUnprocessable(string $message = '', bool ?$verbose = null)``
     Asserts the response is unprocessable (HTTP status is 422)
 
 .. versionadded:: 7.1
 
     The ``$verbose`` parameters were introduced in Symfony 7.1.
 
+.. versionadded:: 7.4
+
+    The ``$defaultVerboseMode = true;`` attribute was introduced in
```

Comment:
This needs some minor reword because it says _"was introduced"_ twice and it's a bit confusing. Thanks.